packages/app-desktop/gui/UpdateNotification/style.scss

```
@@ -1,11 +1,108 @@
+			background: none;
+			border: none;
+			padding: 0;
+			color: inherit;
+			cursor: pointer;
+			text-decoration: underline;
+			text-decoration-color: transparent;
+			transition: opacity 0.2s ease, text-decoration-color 0.2s ease;
+
+			&:hover {
+				opacity: 1;
+				text-decoration-color: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)